config/dev/common-dev.yml

```diff
@@ -1,63 +1,19 @@
-server:
-  forward-headers-strategy: framework
+# This file contains only parameters that needs to be override compare to docker configuration
```

Comment:
that needs => that need
compare to docker configuration => from the docker configuration